src/backend/db/categories.js

```diff
@@ -11,17 +11,20 @@ export const categories = [
     categoryName: "fiction",
     description:
       "literature in the form of prose, especially novels, that describes imaginary events and people",
+    categoryImg: "./assets/categories/men-category.jpg"
   },
```

Comment:
did you add your category? because i didn't see your category

Reply:
Yes will add now, forgot to add